pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: term_glyph
-version: 1.0.0
+version: 1.1.0
```

Comment:
Let's make this a dev version for now. I think we'll probably want to deprecate some of the existing glyphs, since I believe stuff like box-drawing is now completely supported on Windows.